sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiverOptions.cs

```diff
@@ -40,6 +40,13 @@ public int PrefetchCount
         /// </summary>
         public ServiceBusReceiveMode ReceiveMode { get; set; } = ServiceBusReceiveMode.PeekLock;
 
+        /// <summary>
+        /// Gets the ID to identify this client. This can be used to correlate logs and exceptions.
```

Comment:
nit:  We should consider updating this block to remove the implied guarantee about uniqueness, since we don't have control over the value in all cases.